src/metadata.yaml

```diff
@@ -7,6 +7,13 @@ tags:
   - openstack
   - ops
   - monitoring
+links:
```

Comment:
This will not work.  Please review the metadata.yaml spec ( https://juju.is/docs/sdk/metadata-yaml ) and verify that building the charm works.

Reply:
> Starting charmcraft, version 3.1.2    

It might be related to charmcraft v3?  We probably need to pin it to charmcraft v2 (we've started doing it across all our repos gradually).